src/components/Toggle/Toggle.stories.mdx

```diff
@@ -0,0 +1,191 @@
+import {
+  Meta,
+  Story,
+  Canvas,
+  ArgsTable,
+  Description,
+} from "@storybook/addon-docs/blocks";
+import { withDesign } from "storybook-addon-designs";
+import { HStack, VStack, Grid, Box } from "@chakra-ui/react";
+import Heading from "../Heading/Heading";
+import SimpleGrid from "../Grid/SimpleGrid";
+import { getCategory } from "../../utils/componentCategories";
+import DSProvider from "../../theme/provider";
+import Toggle from "./Toggle";
+import { ToggleSizes } from "./ToggleSizes";
+
+<Meta
+  title={getCategory("Toggle")}
+  component={Toggle}
+  decorators={[withDesign]}
+  parameters={{
+    design: {
+      type: "figma",
+      url:
+        "https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Main?node-id=31581%3A302",
+    },
+    jest: ["Toggle.test.tsx"],
+  }}
+  argTypes={{
+    id: { table: { disable: true } },
```

Comment:
Can you do `control: false` instead of `table: { disable: true }`? It's explained more in #788 , but I think it's better to display all props and to disable the control rather than hide it completely. If props are hidden, devs won't know what's available unless they look at the source.